.gitignore

```diff
@@ -1 +1,14 @@
+# Coverage reports
+coverage
+
+# Dependency directory
 node_modules
+
+# OS metadata
+.DS_Store
+
+# Builds
+dist
+
+# Lock files
+yarn.lock
```

Comment:
I think we shouldn't ignore the `yarn.lock` file for the same reason we don't ignore the `package-lock.json` file. Could you remove this line?

Reply:
Work has been updated. Thanks for taking a look at this!